articles/client-platforms/angularjs/01-login.md

```diff
@@ -1,118 +1,240 @@
 ---
-title: Auth0 Angular 1.x Tutorial
-description: This tutorial will show you how to use the Auth0 with Angular 1.x applications.
+title: Login
+description: This tutorial will show you how to use the Auth0 Angular SDK to add authentication and authorization to your mobile app.
 ---
 
-<%= include('../../_includes/_package2', {
-  org: 'auth0-samples',
-  repo: 'auth0-angularjs-sample',
-  path: '00-Starter-Seed'
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-angularjs-sample',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-angularjs-sample',
+  pkgBranch: 'master',
+  pkgPath: '01-Login',
+  pkgFilePath: '01-Login/auth0.variables.js',
+  pkgType: 'replace'
 }) %>
 
-<%= include('_includes/_prerequisite') %>
 
-${include('../\_callback')}
+### 0. Create Auth0 service
 
-#### Create an Application Instance
+The best way to have authentication utilities available across the application is to use an injectable service.
+Notice the `authenticated` success callback which will store the `id_token` in the Local Storage. This value can be retrieved at a later stage.
 
-<%= include('../../_includes/_new_app') %>
+```js
+/* ===== components/auth/auth.service.js ===== */
```

Comment:
for code comment which point to the file path, let's remove the `=====` and also standardize how we do the comment.

For JavaScript files it should be:

`// components/auth/auth.service.js`

and for HTML it should be:

`<!-- components/home/home.template.html -->`